### DISCUSSION TITLE
Bit rate and sample rate

### DISCUSSION THREAD

**Discussion by Gerwin12** [mcguirepr89/BirdNET-Pi#611] [Q&A] [answered]
Hi,

Very inspiring project. I built a birdNET-pi based on raspberry 3B+, simple usb sound card and primo EM172 mono mic. It worked out of the box. Now I am wondering, what is the bit rate and sample rate of the recorder, and is there a way to adjust them? I could not find an answer to this question in any of the discussions.

kind regards,

Gerwin

**Comment by mcguirepr89** [comment, marked as answer]
Hi, @Gerwin12 -- thanks for your kind words about the project and a special thank you for your attempt to find this answer within the discussions. I don't mention anything about the sample-rate within the documentation simply as an oversight, since this is a fork of the BirdNET-Lite repo, where that information is available. I forget, sometimes, that a lot of folks find BirdNET-Pi without ever reaching the BirdNET-Lite github README.md page.

The answer to your question is that the BirdNET-Lite classification model expects (requires) the recording sample rate to be 48kHz. If it is less than that, the model will pad the extra frequency range with noise and will process the audio -- if higher than that sample rate, the audio is down-sampled and whatever was out of range is truncated. For that reason, there are not options for adjusting the sample-rate, as those are set to 48kHz within the recording script.

As far as _bitdepth_ is concerned, the WAVE encoding uses 16-bit Little Endian and is also not adjustable as a setting, though you are welcome to change it in the script `birdnet_recording.sh`.

For the live audio stream's _bitrate_, the audio is encoded _there_ (not sent through analysis) as MP3 and streamed with a 320k bitrate -- the bitrate can be lowered in the script for the service (`livestream.sh`), but there are no options for changing the streaming bitrate. The _bitdepth_ is not set and is automatically chosen by the `libmp3lame` library based on the source (though I have no idea exactly how that goes down, so to speak).

I hope that information is helpful and please accept my best regards~~
Patrick

**Reply by Gerwin12** [reply]
Thank you Patrick, I will check the BirdNET Lite Page for more info, and stay with the 16/48 recording on the BirdNET-Pi. The livestream quality is OK, I really like to listen outside without having to open the window ;-)
